### PR TITLE
v0.15.0 Release Prep

### PR DESCRIPTION
**Pull request by motm32**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
 
 ## Unreleased
 
+## 0.15.0 - 2022-09-06
+
+### Added
+- Support for Virtual Workspaces [#1118] (https://github.com/microsoft/vscode-azurestorage/pull/1118)
```
Comment (alexweininger) [inline CHANGELOG.md:8]: The extra space here is breaking the markdown URLs

```suggestion
- Support for Virtual Workspaces [#1118](https://github.com/microsoft/vscode-azurestorage/pull/1118)
```

**Review by alexweininger — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,23 @@
...
+- Uploaded files to blob containers automatically log file URL's to output [#1076](https://github.com/microsoft/vscode-azurestorage/pull/1076)
+- Activity log support when deleting a folder in a blob container [#1123](https://github.com/microsoft/vscode-azurestorage/pull/1123)
+
+### Changed
+- "Open in File Explorer" command changed to "Open in Explorer" [#1110](https://github.com/microsoft/vscode-azurestorage/pull/1110)
+- Disabled "Open in Storage Explorer" default behavior when in remote workspace [#1109](https://github.com/microsoft/vscode-azurestorage/pull/1109)
+- Minimum VS code version that is supported is now version 1.66.2 [#1106](https://github.com/microsoft/vscode-azurestorage/pull/1106)
+
+### Fixed
+- [Bugs fixed](https://github.com/microsoft/vscode-azurestorage/issues?q=is%3Aclosed+is%3Aissue+milestone%3A0.15.0)
```
Comment (alexweininger) [inline CHANGELOG.md:20]: We fixed some large bugs with attached accounts, we may want to mention something about that here.

Comment (motm32) [reply CHANGELOG.md:20]: Sounds good. I'll add that.

Comment (alexweininger) [reply CHANGELOG.md:20]: The issue was this one, https://github.com/microsoft/vscode-azurestorage/issues/1045 the title doesn't do it justice, it was more than just detaching.

**Review by alexweininger — DISMISSED** [review]

**Review by bwateratmsft — DISMISSED** [review]

**Comment by alexweininger** [comment]
https://github.com/microsoft/vscode-azureresourcegroups/issues/377 needs investigation before release

**Review by bwateratmsft — DISMISSED** [review]

**Review by nturinski — APPROVED** [review]

**Pull request merged**